packages/core/src/hydration/annotate.ts

```diff
@@ -220,6 +221,7 @@ export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
   const viewRefs = appRef._views;
   const shouldReplayEvents = injector.get(IS_EVENT_REPLAY_ENABLED, EVENT_REPLAY_ENABLED_DEFAULT);
   const eventTypesToReplay = new Set<string>();
+  const captureEventTypesToReplay = new Set<string>();
```

Comment:
Since those sets of events are logically connected and would likely be used together (as it happens in this PR, where we've added capture events into the event contract script), I'd propose creating an object here like this:
```suggestion
  const eventTypesToReplay = {
    regular: new Set<string>(),
    capture: new Set<string>(),
  };
```
In this case, the `annotateForHydration` can return an object instead of an array.

Reply:
done